<launch>
	
  <node pkg="simple_arm_control" type="move_right_arm" name="move_arm_to_position" output="screen">
     
     <param name="planner_service_name" value="sbpl_planning/plan_path"/>
		<param name="goal_x" value="0.65"    type="double"/>
	   <param name="goal_y" value="-0.16"  type="double"/>
	   <param name="goal_z" value="0.60"     type="double"/>

	   <param name="goal_roll" value="0"  type="double"/>
	   <param name="goal_yaw" value="0"    type="double"/>
	   <param name="goal_pitch" value="0"  type="double"/>

     <param name="planning_duration" value="20.0" type="double" />
	</node>

</launch>
